src/app/src/main/kotlin/org/vaccineimpact/api/app/models/ChunkedFile.kt

```diff
@@ -0,0 +1,34 @@
+package org.vaccineimpact.api.app.models
+
+import org.vaccineimpact.api.app.ChunkedFileManager.Companion.UPLOAD_DIR
+import java.io.File
+import java.util.concurrent.ConcurrentHashMap
+
+data class ChunkedFile(val totalChunks: Int,
+                       val totalSize: Long,
+                       val chunkSize: Long,
+                       val uniqueIdentifier: String,
+                       val originalFileName: String
+)
+{
+    val uploadedChunks = ConcurrentHashMap<Int, Boolean>()
```

Comment:
So this is a map of ints to bools which are always true if they're there? Could it just be a list of ints and do a contains check? Maybe this way is slightly more efficient though, and it does mean you don't have to do a check before adding the chunk.

Reply:
I just couldn't find a `ConcurrentHashSet` type data structure!